### PR TITLE
KAFKA-7413: Replace slave terminology with follower in website

### PR DESCRIPTION
**Pull request by satybald**
Replace slave terminology with follower in website

Issue [KAFKA-7413](https://issues.apache.org/jira/browse/KAFKA-7413)

### CODE REVIEW

**Comment by vvcephei** [comment]
Hi @satybald ,

Thanks for this. I only have time for a glance right now, but I wanted to drop this thought. It seems like the natural pairs are:
* leader, follower
* primary, replica

I'm not sure whether we say "leader" or "primary" (or something else) elsewhere in this page.

**Comment by satybald** [comment]
@vvcephei thanks for the feedback. I also agree that those pairs are natural pairs. I've updated docs according to your suggestion.

**Comment by vvcephei** [comment]
This looks good to me. Thanks @satybald.

**Comment by ryannedolan** [comment]
nit: "replica" is the copy itself, i.e. the data, not the process doing the replication. Compare with "slave", which is the process not the data, so these terms aren't quite interchangeable. I think "follower" is more appropriate when talking about a process vs the data, as follower implies an agent of some sort (as does slave). Or maybe something like "replica process".

In a couple places you've substituted replica for slave, which I think changes the meaning slightly.

**Review by ijuma — APPROVED** [review]
Thanks for the PR, LGTM.

File: docs/quickstart.html [hunk 1]
```diff
@@ -192,7 +192,7 @@ <h4><a id="quickstart_multibroker" href="#quickstart_multibroker">Step 6: Settin
 &gt; taskkill /pid 6016 /f
 </pre>
 
-<p>Leadership has switched to one of the slaves and node 1 is no longer in the in-sync replica set:</p>
+<p>Leadership has switched to one of the replicas and node 1 is no longer in the in-sync replica set:</p>
```
Comment (ijuma) [inline docs/quickstart.html]: This should be `followers` too right?

**Comment by ijuma** [comment]
Actually, one comment.

**Comment by vvcephei** [comment]
Thanks, @ryannedolan, for the refinement.

I don't know if my anecdotal experience completely lines up with what you say (I'm pretty sure I've heard "replica" referring to a machine or process, not just the actual copy), but if we want to standardize on "primary/replica" for the data and "leader/follower" for the machines/processes, I'm :+1: .

**Comment by satybald** [comment]
@ijuma @vvcephei thank you for a review! I've applied your feedback.

**Review by ijuma — APPROVED** [review]
LGTM

**Pull request merged**